responders/src/UI/embc-responder/src/app/feature-components/wizard/step-create-profile/step-create-profile.component.ts

```diff
@@ -28,6 +35,22 @@ export class StepCreateProfileComponent {
       }
     }
     this.tabs = this.stepCreateProfileService.tabs;
+
+    // Load security question list as soon as wizard is initialized
+    this.securityQuestionsService.getSecurityQuestionList().subscribe(
+      (questions) => {
+        this.stepCreateProfileService.securityQuestionOptions = questions;
+      },
+      (error) => {
+        if (error?.error?.title)
+          this.alertService.setAlert('danger', error.error.title);
+        else
```

Comment:
Custom error messages should be displayed on the UI. Error message can be added to global-constants. Example: AssignedCommunityListComponent